primer/index.bs

```diff
@@ -710,25 +736,29 @@ Token Header:
         "alg": "EC",
         "crv": "P-256",
         "x": "N6VsICiPA1ciAA82Jhv7ykkPL9B0ippUjmla8Snr4HY",
-        "y": "ay9qDOrFGdGe_3hAivW5HnqHYdnYUkXJJevHOBU4z5s",
+        "y": "ay9qDOrFGdGe_3hAivW5HnqHYdnYUkXJJevHOBU4z5s"
     }
 }
 ```
+
 - `"alg": "ES256"`: The signing algorithm used in this token. In this case, `ES256` because we
     generated the keys using eliptic curves.
 - `"typ": "dpop+jwt"`: The type of token. All DPoP Tokens should have a type of "dpop+jwt"
 - `"jwk": { "kty": "EC" ... }`: The client's public key.
 
 Token Body:
+
 ```json
 {
-    "htu": "https://auth.otherpod.example/token",
+    "htu": "https://bob.otherpod.example/private/photo_album.ttl",
```

Comment:
I think this DPoP Proof is used to request Access Token from the `/token` endpoint.
The request to photo_album will happen with a regular Bearer token since it's issued by the AS associated with the RS.

Reply:
@ThisIsMissEm could you please confirm if this change was accidental? And if that's the case revert it

Reply:
Ah, I'd misunderstood the HTU value in the spec — yes, this is an error:
```suggestion
    "htu": "https://auth.otherpod.example/token",
```